### PR TITLE
HPCC-14065  Fix nagios version separator

### PR DESCRIPTION
**Pull request by garonsky**
Signed-off-by: Gleb Aronsky <gleb.aronsky@lexisnexis.com>

### CODE REVIEW

**Comment by garonsky** [comment]
@xwang2713 please review

File: CMakeLists.txt [hunk 1]
```diff
@@ -86,7 +86,7 @@ HPCC_ADD_SUBDIRECTORY( commands )
 MESSAGE (HPCC_SOURCE_DIR: ${HPCC_SOURCE_DIR})
 #ADD_SUBDIRECTORY(${HPCC_SOURCE_DIR} oss EXCLUDE_FROM_ALL)
 
-SET (VER-SEPARTOR "-")
+SET (VER_SEPARTOR "-")
```
Comment (richardkchapman) [inline CMakeLists.txt]: Should this say VER_SEPARATOR?

**Comment by richardkchapman** [comment]
Typo in the commit message (should be separator not seperator)

**Comment by garonsky** [comment]
Fixed typos.

**Pull request merged**